services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala

```diff
@@ -44,16 +44,33 @@ abstract class WorkbenchHealthMonitorServiceActor(val serviceConfig: Config, glo
   }
 
   protected lazy val Gcs = MonitoredSubsystem("GCS", () => checkGcs())
+  protected lazy val CarboniterGcsAccess = MonitoredSubsystem("CarboniterGcsAccess", () => checkGcsAccess(carboniterBucketName, gcsObjectToCheckCarboniterAccess, carboniterGoogleAuth))
   protected lazy val PapiSubsystems = papiBackendConfigurations map papiMonitoredSubsystem
 
   lazy val googleConfig = GoogleConfiguration(globalConfig)
 
   lazy val googleAuthName = serviceConfig.as[Option[String]]("google-auth-name").getOrElse("application-default")
+  lazy val googleAuth = getGoogleAuthConfigurationOrFail(googleAuthName)
 
-  lazy val googleAuth = googleConfig.auth(googleAuthName) match {
-    case Valid(a) => a
-    case Invalid(e) => throw new IllegalArgumentException("Unable to configure WorkbenchHealthMonitor: " + e.toList.mkString(", "))
-  }
+  private lazy val carboniterAuthNameConfigPath = "services.MetadataService.config.carbonite-metadata-service.filesystems.gcs.auth"
+  lazy val carboniterAuthName = getGlobalConfigValueOrFail(carboniterAuthNameConfigPath)
+  lazy val carboniterGoogleAuth = getGoogleAuthConfigurationOrFail(carboniterAuthName)
+
+  private lazy val carboniterBucketNameConfigPath = "services.MetadataService.config.carbonite-metadata-service.bucket"
+  lazy val carboniterBucketName = getGlobalConfigValueOrFail(carboniterBucketNameConfigPath)
+  lazy val gcsObjectToCheckCarboniterAccess = serviceConfig.as[String]("gcs-object-to-check-carboniter-access")
+
+  private def getGlobalConfigValueOrFail(configPath: String): String =
+    globalConfig.as[Option[String]](configPath) match {
```

Comment:
TOL could be `getOrElse`

Reply:
Looks like `getOrElse` here doesn't like `throw` statement as a default value.